src/lib/FederatorERC.ts

```diff
@@ -28,6 +28,7 @@ import {
 } from "../models/failedTransactions.model";
 import {getVote, insertVote} from "../models/votes.model";
 import {insertLogDebug} from "../models/logDebug.model";
+import {log} from "node:util";
```

Comment:
unused import (maybe is worth to add a linter xD)

Reply:
Fixed